bolts/data/datasets/utils.py

```diff
@@ -71,7 +73,7 @@ class BinarySample(NamedTuple):
 
 class TernarySample(NamedTuple):
     x: Tensor | np.ndarray | Image.Image
-    s: Tensor | float
+    z: Tensor | float
```

Comment:
?

Reply:
?

Reply:
Why would you give s such a confusing name?

Reply:
For the very reason that it's not s - it's more general than that. z is hardly an arbitrary choice to go with x and y and indeed it's fairly common in the literature to use it to denote some side information.

Reply:
this will cause problems for `bolts.fair.models` which assume `batch.x`, `batch.s` and `batch.y`